modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,328 @@
+
+| `lb-provider`
+| Optional. Used to specify the provider of the load balancer, for example, `amphora` or `octavia`. Only the Amphora and Octavia providers are supported.
+
+| `lb-version`
+| Optional. The load balancer API version. Only `"v2"` is supported.
+
+| `subnet-id`
+| The ID of the Networking service subnet on which load balancer VIPs are created. 
+
+// This ID is also used to create pool members if `member-subnet-id` is not set.
+
+// | `member-subnet-id`
+// | ID of the Neutron network on which to create the members of the load balancer. The load balancer gets another network port on this subnet. Defaults to `subnet-id` if not set.
+
+| `network-id`
+| The ID of the Networking service network on which load balancer VIPs are created. Unnecessary if `subnet-id` is set.
+
+// | `manage-security-groups`
+// | If the Neutron security groups should be managed separately. Default: false
+
+| `create-monitor`
+| Whether or not to create a health monitor for the service load balancer. A health monitor is required for services that declare `externalTrafficPolicy: Local`. The default value is `false`.
+
+This option is unsupported if you use {rh-openstack} earlier than version 17 with the `ovn` provider.
+
+| `monitor-delay`
+| The interval in seconds by which probes are sent to members of the load balancer. The default value is `5`.
+
+| `monitor-max-retries`
+| The number of successful checks that are required to change the operating status of a load balancer member to `ONLINE`. The valid range is `1` to `10`, and the default value is `1`.
+
+| `monitor-timeout`
+| The time in seconds that a monitor waits to connect to the back end before it times out. The default value is `3`.
+
+| `internal-lb`
+| Whether or not to create an internal load balancer without floating IP addresses. The default value is `false`.
+
+// | `cascade-delete`
+// | Determines whether or not to perform cascade deletion of load balancers. Default: true.
+
+// | `flavor-id`
+// | The id of the loadbalancer flavor to use. Uses octavia default if not set.
+
+// | `availability-zone`
+// | The name of the loadbalancer availability zone to use. It is applicable if use-octavia is set to True and requires Octavia API version 2.14 or later (Ussuri release). The Octavia availability zone capabilities will not be used if it is not set. The parameter will be ignored if the Octavia version doesn't support availability zones yet.
+
+| `LoadBalancerClass "ClassName"`
+a| This is a config section that comprises a set of options:
+
+ * `floating-network-id`
+ * `floating-subnet-id`
+ * `floating-subnet`
+ * `floating-subnet-tags`
+ * `network-id`
+ * `subnet-id`
+
+//  * `member-subnet-id`
+
+The behavior of these options is the same as that of the identically named options in the load balancer section of the CCM config file.
+
+You can set the `ClassName` value by specifying the Service annotation `loadbalancer.openstack.org/class`. 
```

Comment:
Here and in line 236, should "Service" be lowercase?

Reply:
```suggestion
You can set the `ClassName` value by specifying the service annotation `loadbalancer.openstack.org/class`. 
```